### PR TITLE
Add the uid-/gidnumber as an argument to ipa_user

### PR DESCRIPTION
**Pull request by smartsammler**
##### SUMMARY
Fixes issue #32555

* Adds the uidnumber and the gidnumber which are called UID and GID in IPA's web interface, but are called `uidnumber` and `gidnumber` in IPA's JSON API and therefore in ansible's `ipa_user`s return value.
* To avoid confusion with `ipa_user`'s `uid` I stayed with the JSON API names of the `user_mod` command.
* I changed the code analogue to PR #32369

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

ipa_user

##### ANSIBLE VERSION

```
N/A
```

##### ADDITIONAL INFORMATION
I tested it locally with the following setup:
IPA client:
--------------
Fedora 25 workstation 
```
$ ipa --version
VERSION: 4.4.4, API_VERSION: 2.215
```

IPA server: 
----------------
CentOS 7
```
$ ipa --version
 VERSION: 4.5.0, API_VERSION: 2.228
```

ansible role
------------------
``` yaml
---
- name: activate user
  ipa_user:
    ipa_host: my_ipa_server.tld
    ipa_pass: top_secret
    uid: testuser
    state: enabled
    uidnumber: 1030
    gidnumber: 1030
```

### CODE REVIEW

File: lib/ansible/modules/identity/ipa/ipa_user.py [hunk 1]
```diff
@@ -49,12 +49,21 @@
     - List of telephone numbers assigned to the user.
     - If an empty list is passed all assigned telephone numbers will be deleted.
     - If None is passed telephone numbers will not be checked or changed.
+    version_added: 2.5
```
Comment (Akasurde) [inline lib/ansible/modules/identity/ipa/ipa_user.py]: Telephonenumber is already added in version 2.3.

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

File: lib/ansible/modules/identity/ipa/ipa_user.py (hunk 1 above)
Comment (smartsammler) [reply lib/ansible/modules/identity/ipa/ipa_user.py]: Uups. Sorry, should be fixed now.

**Review by Akasurde — APPROVED** [review]
shipit

File: lib/ansible/modules/identity/ipa/ipa_user.py [hunk 2]
```diff
@@ -217,7 +231,8 @@ def ensure(module, client):
                                 mail=module.params['mail'], sn=module.params['sn'],
                                 sshpubkey=module.params['sshpubkey'], nsaccountlock=nsaccountlock,
                                 telephonenumber=module.params['telephonenumber'], title=module.params['title'],
-                                userpassword=module.params['password'])
+                                userpassword=module.params['password'],
+                                gidnumber=module.params.get('gidnumber'), uidnumber=module.params.get('uidnumber'))
```
Comment (resmo) [inline lib/ansible/modules/identity/ipa/ipa_user.py]: would you mind creating 2 lines here? for each param?

File: lib/ansible/modules/identity/ipa/ipa_user.py [hunk 3]
```diff
@@ -250,6 +265,7 @@ def main():
                          mail=dict(type='list'),
                          sn=dict(type='str'),
                          uid=dict(type='str', required=True, aliases=['name']),
+                         gidnumber=dict(type='str'), uidnumber=dict(type='str'),
```
Comment (resmo) [inline lib/ansible/modules/identity/ipa/ipa_user.py]: would you mind creating 2 lines here? for each param?

Comment (smartsammler) [reply lib/ansible/modules/identity/ipa/ipa_user.py]: done

**Pull request merged**